public/app/features/plugins/sandbox/utils.ts

```diff
@@ -7,3 +9,10 @@ export function isSandboxedPluginObject(value: unknown): value is SandboxedPlugi
 export function assertNever(x: never): never {
   throw new Error(`Unexpected object: ${x}. This should never happen.`);
 }
+
+export function isReactClassComponent(obj: unknown): obj is React.Component {
+  if (obj instanceof React.Component) {
+    return true;
+  }
+  return false;
```

Comment:
maybe here you could return the result of instance of directly:
```ts
  return obj instanceof React.Component
```
since it's a boolean

Reply:
Indeed. I was initially doing more checks here and ended up in this simplified logic. Thanks for the suggestions.